reconcile/utils/jump_host.py

```diff
@@ -71,3 +78,25 @@ def get_ssh_base_cmd(self):
             '-o', 'StrictHostKeyChecking=yes',
             '-o', 'UserKnownHostsFile={}'.format(self.known_hosts_file),
             '-i', self.identity_file, '-p', str(self.port), user_host]
+
+    def create_ssh_tunnel(self, local_port, remote_port):
+        key = f'{self.hostname}-{local_port}-{remote_port}'
+        JumpHostSSH.tunnel_lock.acquire()
```

Comment:
nit: why not `self.tunnel_lock....` ?

Reply:
Both the locks and the tunnel info needs to exist across multiple different instantiations of jump host to avoid tunnels being opened simultaneously or to hosts that already have a tunnel.

Reply:
:facepalm: totally